etc/events.xml

```diff
@@ -1,5 +1,44 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
     <event name="clean_cache_by_tags">
-        <observer name="invalidate_cloudflare_observer" instance="CloudFlare\Plugin\Observer\InvalidateCloudFlareObserver"/>
```

Comment:
Now that we have both purge cache observers we can mimic all the events they hook into for varnish.
https://github.com/magento/magento2/blob/6ea7d2d85cded3fa0fbcf4e7aa0dcd4edbf568a6/app/code/Magento/CacheInvalidate/etc/events.xml